test/mlir/ngraph_dialect/pattern/fused_ops.mlir

```diff
@@ -0,0 +1,19 @@
+// RUN: ngraph-opt %s -fuse-ngraph-dialect -split-input-file | FileCheck %s
+
+// Verify that operations fused using pattern matcher are properly replaced with correct Fused Op.
+
+// -----
+
+// matmul+bias
+// CHECK-LABEL: module {
+// CHECK-LABEL:   func @matmul_bias_fusion(%arg0: !ng.tensor<2x4xf32>, %arg1: !ng.tensor<4x1xf32>, %arg2: !ng.tensor<2x1xf32>) -> !ng.tensor<2x1xf32> {
+//       CHECK:   %0 = "ng.gemm"(%arg0, %arg1, %arg2) {alpha = 1.000000e+00 : f32, beta = 1.000000e+00 : f32, transA = false, transB = false} : (!ng.tensor<2x4xf32>, !ng.tensor<4x1xf32>, !ng.tensor<2x1xf32>) -> !ng.tensor<2x1xf32>
+//       CHECK:   "ng.return"(%0) : (!ng.tensor<2x1xf32>) -> ()
+// 	 CHECK:   }
+// 	 CHECK: }
```

Comment:
It looks great! Some minor comments so that the test does't break with minor variations of the MLIR repo:
    - Please remove line 8, 12 and 13 since they are not needed to test this feature.
    - Please capture `%0` and `%argX`s with a regular expression in lines 10 and 11.